agents/check_mk_agent.macosx

```diff
@@ -192,11 +193,42 @@ tr -d ' '` | bc
 # checks plugins with subchecks for parsing that output. Maybe reduce
 # the output size by grepping away totally useless parts
 
-echo '<<<netctr>>>';
```

Comment:
Unfortunately (and the reason for this PR), is that netctr was deprecated but noone updated this agent! So deleting this and replacing with lnx_if as below so that I can monitor interfaces of my mac mini servers.